config/pmd.xml

```diff
@@ -130,6 +164,8 @@
     <exclude name="GodClass"/>
     <!-- Too many violations, will be suppressed until we find out how use these metrics. -->
     <exclude name="LawOfDemeter"/>
+    <!-- Does not work out of the box. Must be configured in an inconvenient way. -->
+    <exclude name="LoosePackageCoupling"/>
```

Comment:
I do not see that we excluded it before. Please warn us on new configuration (not moved).
http://pmd.sourceforge.net/pmd-5.0.1/rules/java/coupling.html

comment should be:
"we use our ImportControl to control imports in packages and classes"

Reply:
Done. The rule produces the warning
```
[WARNING] Removed misconfigured rule: LoosePackageCoupling  cause: No packages or classes specified
```
So it must be either configured or excluded.

Reply:
thanks for explanation.